### DISCUSSION TITLE
Copy fields?

### DISCUSSION THREAD

**Discussion by xcellentavi** [docusealco/docuseal#234] [Q&A] [answered]
Is there a way to copy a field and paste it on different pages (or multiple places in the same page)?  I know there is a "Copy to All Pages" option in the settings section on the right side of the field, but is there other ways to do this?  Thank you!

**Comment by omohokcoj** [comment, marked as answer]
Hi @xcellentavi it's possible to drag and drop existing field on the page to add it on a new position/page. Also you can draw the same field with a 'draw new area' option from the settings. Please let me know if it answers your question
![image](https://github.com/docusealco/docuseal/assets/5418788/13cec683-3561-4505-92d3-57ab4c1975a8)

**Reply by xcellentavi** [reply]
Thank you @omohokcoj, I don't know why I didn't try to drag and drop.  Maybe that didn't work for me in earlier versions, but this is perfect.  Now I just need to figure out how to delete archived items and I should be all set.
